open_rarity/models/token.py

```diff
@@ -69,5 +70,26 @@ class           attribute type
             metadata=TokenMetadata.from_attributes(metadata_dict),
         )
 
+    @classmethod
+    def from_dict(cls, data_dict: dict):
+        identifier_class = get_identifier_class_from_dict(
+            data_dict["token_identifier"]
+        )
+
+        return cls(
+            token_identifier=identifier_class.from_dict(
+                data_dict["token_identifier"]
+            ),
+            token_standard=TokenStandard[data_dict["token_standard"]],
```

Comment:
Shouldn't this be `TokenStandard(data_dict["token_standard"])`

Reply:
No use `()` for enum's value for `[]` for enum's name haha